app/controllers/api/v3/analytics/user_analytics_controller.rb

```diff
@@ -1,13 +1,20 @@
 class Api::V3::Analytics::UserAnalyticsController < Api::V3::AnalyticsController
   include ApplicationHelper
+  before_action :set_for_end_of_month
 
   layout false
 
   def show
     @user_analytics = UserAnalyticsPresenter.new(current_facility, with_exclusions: report_with_exclusions?)
+
+    @protocol_drugs = @current_facility.protocol.protocol_drugs.where(stock_tracked: true).sort_by(&:sort_key)
```

Comment:
Can we extract this sorted list into a model? It can be reused in the drug stock controller as well.